### DISCUSSION TITLE
Inverting letters tft35 Help mne please:(:(

### DISCUSSION THREAD

**Discussion by sokolek155** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2412] [Q&A]
![20220219_125311](https://user-images.githubusercontent.com/84595174/154802435-9fd22561-dcae-448a-aafc-9ce60afb58e1.jpg)
 
Letters reversed when updating firmware in tft35 v3. Changing the config.ini file does not help.

**Comment by artineo** [comment]
Don't know, but maybe You choose BIGTREE_GD_TFT35_V3.0.27.x.bin (or similar) , try without GD.
